Packs/Cybereason/Playbooks/playbook-Unisolate_Endpoint_-_Cybereason.yml

```diff
@@ -25,7 +25,6 @@ tasks:
     ignoreworker: false
     skipunavailable: false
```

Comment:
Did you include this change? Do we need it?
If it's not necessary, please revert it.

Reply:
Hi, There were some demisto validation errors for this Playbook. So, just fixed those errors.